books/free-programming-books-subjects.md

```diff
@@ -856,6 +856,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [How to deal with Passwords](https://github.com/MHM5000/pass)
 * [Information Security Management](https://www.infosecuritybook.com) - Marcos Sêmola (PDF)
 * [Intrusion Detection Systems with Snort](https://ptgmedia.pearsoncmg.com/images/0131407333/downloads/0131407333.pdf) (PDF)
+* [MoonMath - zk-SNARKs](https://leastauthority.com/community-matters/moonmath-manual/) - Least Authority
```

Comment:
```suggestion
* [The MooonMath Manual to zk-SNARKs](https://leastauthority.com/community-matters/moonmath-manual/) - Least Authority
```
Use the title given by the resource. (understanding that the resource uses several varations of the title, I don't think it uses the one you've given.) Will need to reorder it in the list. Super addition, though.

Reply:
Thanks, force-pushed 2fcbc672e9df8436bc0a50adf4f497f01ffd3c6c to adopt the suggestion. (Force-pushed twice because the first time I forgot to reorder the list.)